docs/CLI-Reference.md

```diff
@@ -146,6 +146,9 @@ This file is auto-generated from the output of `cl help -v` and provides the lis
           search .limit=<limit>                  : Limit the number of results to the top <limit> (e.g., 50).
           search .offset=<offset>                : Return results starting at <offset>.
         
+          search .before=<datetime>              : Returns bundles created before given ISO 8601 timestamp (exclusive).
```

Comment:
I think you need to regenerate this doc, it seems to be out of sync with bundle_cli.py

Reply:
Should we add a git hook for this (generating docs)? @percyliang can I open a bug on myself for this?

Reply:
I think this is already run in `pre-commit.sh`.